packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/ToggleSwitch.tsx

```diff
@@ -50,17 +49,28 @@ export interface ToggleSwitchProps extends Props {
  * [GitHub](https://github.com/guardian/source/tree/main/packages/@guardian/source-react-components-development-kitchen/components/toggle-switch) •
  * [NPM](https://www.npmjs.com/package/@guardian/source-react-components-development-kitchen)
  *
- * Displays an on/off toggle switch. This toggle has default styling and can be used in andriod or ios.
+ * Displays an on/off toggle switch. This toggle has default styling and can be used on andriod, ios or web.
+ * These styles are driven by the 'platform' prop.
  * To give it more custome styling cssOverride may be used.
  *
  */
+const getPlatformStyles = (platform: Platform): SerializedStyles => {
+	switch (platform) {
+		case 'android':
+			return androidStyles;
+		case 'ios':
+			return iosStyles;
+		default:
```

Comment:
would this be safer like this?
```suggestion
		case 'web':
```
`css={[buttonStyles, undefined]}` feels like a more accurate alert that something has gone wrong in the code (rather than just using `web` styles if something has gone wrong)?

Reply:
Yeah good call.